### PR TITLE
Amplitude group call update

### PR DESCRIPTION
**Pull request by ameypv-rudder**
## Description of the change

> Updated `group` section in Amplitude to clarify multiple user group support.

## Type of documentation

- [ ] New documentation
- [x] Documentation update
- [ ] Hotfix

## Notion ticket link

> [JS SDK - Group call (support for multiple groups per user?)](https://www.notion.so/rudderstacks/JS-SDK-Group-call-support-for-multiple-groups-per-user-ca0d38ffcea8490dbfbfca72f1c663eb)

### CODE REVIEW

File: docs/destinations/analytics/amplitude.mdx [hunk 1]
```diff
@@ -535,23 +535,25 @@ A sample dashboard after making the above `identify`, `page`, and `track` calls
...
-Groups are an enterprise-only feature in Amplitude and you need to purchase the Accounts add-on to use them.
+<div class="infoBlock">
+When sending events via the device mode, <code class="inline-code">group</code> is currently supported only by the RudderStack JavaScript SDK.
 </div>
 
-The `group` call lets you associate a particular identified user with a group, such as a company, organization, or an account.
+To use the Amplitude Groups feature with RudderStack, you need to define the <strong>Group Name Trait</strong> and <strong>Group Value Trait</strong> in the dashboard settings and pass them as traits while making a `group` call.
 
-In order to use the Amplitude groups feature with RudderStack you need to define the `Group name trait` and `Group value trait` on the dashboard settings pane and you need to pass them as traits while making a group call.
+<div class="warningBlock">
```
Comment (utsabc) [inline docs/destinations/analytics/amplitude.mdx:550]: We can make it an info rather than warning

Comment (ameypv-rudder) [inline docs/destinations/analytics/amplitude.mdx:550]:
```suggestion
<div class="infoBlock">
```

File: docs/stream-sources/rudderstack-sdk-integration-guides/rudderstack-javascript-sdk/index.mdx [hunk 2]
```diff
@@ -455,6 +455,10 @@ rudderanalytics.alias("test_new_id", "old_user_id", () => {
 
 The `group` call lets you link an identified user with a group such as a company, organization, or an account. It also lets you record any custom traits associated with that group such as the name of the company, the number of employees, etc.
 
+<div class="warningBlock">
```
Comment (ameypv-rudder) [inline docs/stream-sources/rudderstack-sdk-integration-guides/rudderstack-javascript-sdk/index.mdx:458]:
```suggestion
<div class="infoBlock">
```

File: docs/destinations/analytics/amplitude.mdx (hunk 1 above)
Comment (ameypv-rudder) [reply docs/destinations/analytics/amplitude.mdx]: Sure - this is done.

File: docs/destinations/analytics/amplitude.mdx [hunk 3]
```diff
@@ -535,23 +535,25 @@ A sample dashboard after making the above `identify`, `page`, and `track` calls
...
+</div>
 
-Even if you are not having an enterprise account or groups add-on, RudderStack would add `groups`as a `User Property` on a user's profile with `Group name trait` as its type and `Group value trait` as its value.
+Even if you don't have an enterprise account or the Groups add-on, RudderStack adds `groups` as a user property in the user's profile with <strong>Group Name Trait</strong> as its type and <strong>Group Value Trait</strong> as its value.
 
-Let' say you had defined `Group name trait` as `company_id` and `Group value trait` as `company_name` and made the above call then the user would be associated with a Group name : `RS` and a Group Value : `RudderStack` and you can also see that a user property with type `RS` and value `RudderStack` on user's profile.
+Suppose you defined <strong>Group Name Trait</strong> as `company_id` and <strong>Group Value Trait</strong> as `company_name` and made the `group` call. Then, the user would be associated with a Group name : `RS` and a Group Value : `RudderStack` and you can also see the user property with type `RS` and value `RudderStack` in the user profile.
```
Comment (ishachopra0401) [inline docs/destinations/analytics/amplitude.mdx:556]:
```suggestion
Suppose you defined the <strong>Group Name Trait</strong> as `company_id` and <strong>Group Value Trait</strong> as `company_name` and made the `group` call, then, the user would be associated with the Group name : `RS` and the Group Value : `RudderStack`. You can also see the user property with type `RS` and value `RudderStack` in the user profile.
```

1) Possibly:
Group Name Trait as RS
Group Value Trait as RudderStack

2) last line: user property with **type** RS and value RudderStack in the user profile.
should **type** -> **name** instead?

File: docs/destinations/analytics/amplitude.mdx [hunk 4]
```diff
@@ -528,30 +507,30 @@ A sample dashboard after making the above `identify`, `page`, and `track` calls
...
+</div>
 
-Even if you are not having an enterprise account or groups add-on, RudderStack would add `groups`as a `User Property` on a user's profile with `Group name trait` as its type and `Group value trait` as its value.
+Even if you don't have an enterprise account or the Groups add-on, RudderStack adds `groups` as a user property in the user's profile with <strong>Group Name Trait</strong> as its type and <strong>Group Value Trait</strong> as its value.
 
-Let' say you had defined `Group name trait` as `company_id` and `Group value trait` as `company_name` and made the above call then the user would be associated with a Group name : `RS` and a Group Value : `RudderStack` and you can also see that a user property with type `RS` and value `RudderStack` on user's profile.
+Suppose you defined <strong>Group Name Trait</strong> as `RS` and <strong>Group Value Trait</strong> as `RudderStack` and made the `group` call. Then, the user would be associated with a Group name : `RS` and a Group Value : `RudderStack`.
```
Comment (ishachopra0401) [inline docs/destinations/analytics/amplitude.mdx:533]:
```suggestion
Suppose you have defined the <strong>Group Name Trait</strong> as `RS` and <strong>Group Value Trait</strong> as `RudderStack` and made the `group` call, then the user would be associated with the Group name: `RS` and the Group Value: `RudderStack`.
```

**Review by ishachopra0401 — APPROVED** [review]

**Pull request merged**